requirements.txt

```diff
@@ -1,4 +1,4 @@
-dwave-ocean-sdk>4.2.0
+dwave-ocean-sdk>4.3.0
```

Comment:
Ah, this should be: `>=4.3.0`

Reply:
Yes, I saw that and force pushed it already